README.md

```diff
@@ -37,6 +37,12 @@ this extension does not automatically _fix_ lint warnings during formatting,
 but you can opt into this by enabling the **Bazel: Buildifier Fix on Format**
 setting.
 
+### Limiting quick pick queries to the nearest package
```

Comment:
(opinionated) I feel like the separate output base is a configuration that will affect more users then the limiting of quick pick queries. Hence I would place the new config option below the existing one.